package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@cowprotocol/cow-sdk",
-  "version": "3.0.1",
+  "version": "3.0.2",
```

Comment:
mmm, will this be a breaking change for SDK clients?

Reply:
Sure, bumped to major semver.